src/route/reset.route.ts

```diff
@@ -34,22 +29,24 @@ resetRouter.delete("/", authorizeUser, async (req: Request, res: Response) => {
           "a59e5585c1b2b33a91a25ededf39827f93996da9b912f26f9766d45f9dfb0742",
       },
     });
+
     //await defaultuser.save();
     if (res.locals.isAdmin) {
+      logger.info("Resetting registry")
       await PackageModel.deleteMany({});
       await ProfileModel.deleteMany({});
+      await PackageRatingModel.deleteMany({});
```

Comment:
What does this do?

Reply:
We save the ratings in our database so that we don't have to rerate a module everytime they hit the rate endpoint. So in /package/rate, we first check if the package has a rating or not before rating it. I assume in the default state we shouldn't save any cached ratings though.